fields/types/Field.js

```diff
@@ -69,8 +69,9 @@ var Base = module.exports.Base = {
     return <FormInput noedit>{this.props.value}</FormInput>;
   },
   renderUI() {
-    var wrapperClassName = classnames(
-      'field-type-' + this.props.type,
+    const field = this.props.field || this.props.type;
```

Comment:
why is the field default to `this.props.type`?

Reply:
I think this.props.field is a redundant prop, will remove it along with Ln 18 in https://github.com/twreporter/plate/pull/263.